services/user-svc/main.go

```diff
@@ -28,13 +28,19 @@ func main() {
 	closeDBPool := hwdb.SetupDatabaseFromEnv(ctx)
 	defer closeDBPool()
 
-	common.StartNewGRPCServer(ctx, common.ResolveAddrFromEnv(), func(server *daprd.Server) {
-		grpcServer := server.GrpcServer()
-
-		kc, err := hwkc.NewClientFromEnv(ctx)
+	var kc hwkc.IClient
+	if common.Mode == common.DevelopmentMode {
```

Comment:
this should not be determined by mode, and I also dont think it should be determined in main()

maybe introduce another env variable and hint to it when failing to create the real client (because of missing variables)

Reply:
should start with INSECURE_

Reply:
also, sometimes new clients are created in handlers, and other times multiple handlers use one client

in that last case: does that underlying shared http client cause any problems wrt concurrency?